.pylintrc

```diff
@@ -93,6 +93,7 @@ disable=fixme,
         too-few-public-methods,
         too-many-arguments,
         too-many-locals,
+        too-many-ancestors,
```

Comment:
why? this is encouraging horrifying inheritance patterns

Reply:
Pylint confuses implementing interfaces (protocols) with inheritance.

Reply:
I think disabling such rule should be discussed prior with other team members and done in a separate PR

Reply:
IMO it's an obvious linter bug caused by hacky support of interfaces in Python.